src/client.ts

```diff
@@ -6,6 +6,16 @@ import { Credentials, fromStaticCredentials } from "./credentials.ts";
 import type { AwsErrorMeta } from "./error.ts";
 import { DefaultFetch, Fetch } from "./fetch.service.ts";
 import type { ProtocolHandler } from "./protocols/interface.ts";
+import * as Schedule from "effect/Schedule";
+
+export const retryableErrorTags = [
+  "InternalFailure",
+  "RequestExpired",
+  "ServiceException",
+  "ServiceUnavailable",
+  "ThrottlingException",
+  "TooManyRequestsException",
+];
```

Comment:
Is this all of the errors? Can we extract them. Maybe the smithy models has information we cam use, e.g. all 5xx errors should be retried. Maybe there's other metadata.